README.md

```diff
@@ -0,0 +1,45 @@
+# cypress-utils-criteo
+
+This packages offers custom Cypress commands, recommended checks, and Angular Material selectors to ease the development of Cypress tests.
```

Comment:
attributes selectors? 
Is it really related to Angular Material?

Reply:
I meant selectors and functions specific to Angular Material (dropdowns, snack bars, slide toggles,...) How would you call it?

Reply:
Yes indeed, I guess this is the right name